doc/compodoc_sources/how-to-guides/reports.md

```diff
@@ -16,6 +16,38 @@ SQS enables the possibility to create SQL based queries in reports.
 
 The SQS image is not available as open source and needs a licence. It is pulled from a private container registry.
 
+## Using Copilot to generate SQL Queries
+
+```json
+// copy the full Config:CONFIG_ENTITY doc from the database
+// and use the following code to extract the entities as context for Copilot
+entities = Object.entries(x.data).filter(([k, v]) => k.startsWith("entity:")).map(([k, v]) => v);
+```
+
+Use the following prompt sample:
```

Comment:
maybe we can also add/mention that this is only for the v1, and later we can add a snippet for v2 as well?